pkg/controller/seed/reconciler.go

```diff
@@ -80,6 +80,16 @@ func (kcr *kubeSystemReconciler) reconcile(ctx context.Context, logger logr.Logg
 	const (
 		kubeSystemNamespaceName = metav1.NamespaceSystem
 	)
+	var (
+		ownerNamespace = kcr.serviceConfig.SeedBootstrap.OwnerNamespace
+	)
+
+	if !kcr.serviceConfig.DeploySeedResources {
+		if err := managedresources.DeleteForSeed(ctx, kcr.client, ownerNamespace, constants.ManagedResourceNamesSeed); err != nil {
```

Comment:
Let's add log messages showing when the admission webhook is being uninstalled, and when installed.

```suggestion
	if !kcr.serviceConfig.DeploySeedResources {
		logger.Info("Deleting lakom admission controller from the seed cluster")
		if err := managedresources.DeleteForSeed(ctx, kcr.client, ownerNamespace, constants.ManagedResourceNamesSeed); err != nil {
```